doc/classes/Basis.xml

```diff
@@ -247,7 +267,7 @@
 			<return type="Basis" />
 			<param index="0" name="scale" type="Vector3" />
 			<description>
-				Introduce an additional scaling specified by the given 3D scaling factor.
+				Returns this basis with each axis' components scaled by the given [param scale]'s components.
```

Comment:
We're getting back to the territory of "I'm not sure most users need this information". Those that are heavily interested in the Basis's rows would have already inferred this and have their doubts "quenched" by the example below (but perhaps it doesn't hurt to repeat it).

Also, "global or parent-relative scale" does not feel like a concept that applies to a standalone Basis, and because of this I'm finding difficult to integrate it nicely.

Reply:
Basically by "global or parent-relative scale" I am trying to convey the concept of a transform being applied above/before/from the left/etc, not below/after/from the right/etc as a child of the basis.

Reply:
Wouldn't this be more appropriate to describe in **Transform3D**'s class reference, instead? I will keep it in mind still, because **Transform3D**'s docs make similar claims.

Reply:
Well, a Basis is still a linear transformation, just with only the rotation/scale/shear/etc and no position.